src/Signature/SignatureV4.php

```diff
@@ -82,7 +82,9 @@ public function presign(
         $expires,
         array $options = []
     ) {
-        $startTimestamp = isset($options['start_time']) ? $this->convertToTimestamp($options['start_time']) : time();
+
+        $startTimestamp = isset($options['start_time']) ? $this->convertToTimestamp($options['start_time'], null) : time();
```

Comment:
An example of what our 3 line ternary looks like [can be found here.](https://github.com/aws/aws-sdk-php/blob/master/src/S3/BatchDelete.php#L170)